### DISCUSSION TITLE
Modelfile Content of GGUF to add a custom system prompt?

### DISCUSSION THREAD

**Discussion by Immortality-IMT** [open-webui/open-webui#3693] [Q&A]
Settings: Models: Experimental: Upload a GGUF then displays Modelfile content with...

TEMPLATE "here"{{ .System }}
USER: "or here" {{ .Prompt }}
ASSISTANT: """
PARAMETER num_ctx 4096
PARAMETER stop "</s>"
PARAMETER stop "USER:"
PARAMETER stop "ASSISTANT:"

This is editable, any way to inject a pre-prompt to every inference, maybe TEMPLATE "here"{{ .System }} or USER: "or here" {{ .Prompt }}

Thanks in advance, any help appreciated.

**Comment by antonio-lamberti** [comment]
Did you solve this?

**Comment by Immortality-IMT** [comment]
Yes, the latest open-webui has the ability to custom prompt each model via Workspace
